packages/fast-tooling-react/src/css-property-editor/property-editor.spec.tsx

```diff
@@ -144,4 +137,43 @@ describe("CSSPropertyEditor", () => {
             shallow(<CSSPropertyEditor data={data} />);
         }).not.toThrow();
     });
+
+    test("row with empty key gets delected on input blur", () => {
```

Comment:
can you change this text to be "should remove the active row with empty key on input blur"

Reply:
yep